guides/common/modules/proc_configuring-a-host-for-registration.adoc

```diff
@@ -23,13 +23,6 @@ ifdef::client-content-dnf[]
 ----
```

Comment:
Just above this is the "On {EL} 7 and later" header. Is that header still relevant now or can we drop it instead?

Reply:
I suggest to reword line 21 to "On {EL}:". What do you think Lena?